docs/development/pigs/pig-026.rst

```diff
@@ -0,0 +1,91 @@
+.. include:: ../../references.txt
+
+.. _pig-026:
+
+********************************
+PIG 26 - Priors and FitStatistic
+********************************
+
+* Author: Noah Biederbeck
+* Created: ``2023-03-13``
+* Accepted:
+* Status: draft
+* Discussion: `#4381`_
+
+Abstract
+========
+
+This PIG is intended to introduce priors on models that are evaluated during fitting.
+
+Motivation
+==========
+
+Using priors on models and on parameters is the next step in full-fledged statistical analyses.
+A prior can incorporate the analysers' knowledge of the topic at hand, and yield more realistically and trustworthy results.
+The proposed formalism also includes application of nuisance parameters and regularization.
+
+Use cases
+=========
+
+In the past priors were a regularly requested feature or the solution to a problem.
+See the following issues and PRs of Gammapy:
+
+- Add sigma v estimator functionality for dark matter use case `#2075`_
+- Background systematics as a nuisance parameter `#3955`_
+- Support unfolding methods for spectral flux points `#4122`_
+- Add PiecewiseNormSpatialModel `#4208`_
+
+Implementation
+==============
+
+See `#4237`_ for an implementing PR draft.
+
+Priors on parameters:
+
+.. code:: python
+
+    class GaussianPrior():
+        def __init__(self, mu=0, sigma=1):
+            self.mu = mu
+            self.sigma = sigma
+
+        def stat_sum(self, value):
+            return ((value - self.mu) / self.sigma) ** 2
+
+    dataset.models.parameters["x"].prior = GaussianPrior()
+
+
+Priors on models:
+
+.. code:: python
+
+    class L2Regularization():
```

Comment:
Note that this case actually corresponds to the `GaussianPrior` on each of the individual parameters, where `mu=0 and sigma=1`. The most general case I could imagine would be `MultiVariateGaussianPrior` on a list of parameters. The case of `L2` regularization would be included there...but maybe we just keep it for convenience.